setup.cfg

```diff
@@ -1,2 +1,15 @@
 [flake8]
-max-line-length = 99
\ No newline at end of file
+max-line-length = 99
```

Comment:
the norm for max-line-length is usuall 79. why 99?

Reply:
I believe that @romanlutz  is too young to have used F77, and I only did so under duress :-)

Reply:
@bethz if you only have a screen that accommodates 79 characters I'll be happy to talk to facilities for you :D 
It's simply annoying to have this restriction. Code becomes unreadable if you constantly have to wrap it around. I'm actually for upping this to 119

Reply:
I'm not sure I'd want 119 for everything, but allowing a few longer lines per source file (without having to put a special exception in) would be a nice feature for the line-length check.